Makefile

```diff
@@ -1,11 +1,14 @@
 .PHONY: build
 
-install:
+install: package.json
 	@npm install
 
-run:
+run: .example_install
```

Comment:
For no particular reason. `make` keeps calling `npm install` on my laptop even though the `package.json` didn't change. It read examples with a leading dot, so I thought it could be significant, but I think it's just BSD's `make` that behaves differently.

I removed the dot.